### DISCUSSION TITLE
Three.js with sli.dev!!!!

### DISCUSSION THREAD

**Discussion by Leone25** [slidevjs/slidev#680] [Show and tell]
Hello,

today I'd like to share something I made for a school presentation. (I removed all the original text)
Link: https://slidevthreejs.enrico.wtf/
(It's slow, ik)

It's made with sli.dev + a bottom global layer with three.js on it.
Code: https://github.com/Leone25/slidev-threejs

My implementation is awful, let's be clear, but someone might get inspired and turn this in to a complete project/feature. After all, I worked on it for 2 weeks in my free time. The animations are all made by hand, using three.js tools to move stuff around, mark down positions and then copying them over to the json. The models are taken from robosim by comau, a software I was using during the course I followed that lead up to this presentation.

I'll be happy to answer any questions.

I hope you enjoy.

Enrico

**Comment by ahandsel** [comment]
@Leone25 Cool slides 🤖
